### PR TITLE
fix: add missing eslint import in generated package.json

### LINKED ISSUE #374: ESLint config imports a transitive dependency that is not installed (pnpm 10)
We import `@eslint/js` but never actually add it to `package.json`, so linting actually fails even on a minimal fresh setup:
```
$ pnpx sv create sample
┌  Welcome to the Svelte CLI! (v0.6.9)
│
◇  Which template would you like?
│  SvelteKit minimal
│
◇  Add type checking with Typescript?
│  No
│
◆  Project created
│
◇  What would you like to add to your project? (use arrow keys / space bar)
│  eslint
│
◇  Which package manager do you want to install dependencies with?
│  pnpm
│
◆  Successfully setup add-ons
│
◆  Successfully installed dependencies
│
◇  Project next steps ─────────────────────────────────────────────────────╮
│                                                                          │
│  1: cd sample                                                            │
│  2: git init && git add -A && git commit -m "Initial commit" (optional)  │
│  3: pnpm dev --open                                                      │
│                                                                          │
│  To close the dev server, hit Ctrl-C                                     │
│                                                                          │
│  Stuck? Visit us at https://svelte.dev/chat                              │
│                                                                          │
├──────────────────────────────────────────────────────────────────────────╯
│
└  You're all set!
$ cd sample
$ pnpm lint

> sample@0.0.1 lint C:\Users\glitterforce\Documents\sample
> eslint .


Oops! Something went wrong! :(

ESLint: 9.17.0

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@eslint/js' imported from C:\Users\glitterforce\Documents\sample\eslint.config.js
Did you mean to import "@eslint/js/src/index.js"?
    at packageResolve (node:internal/modules/esm/resolve:857:9)
    at moduleResolve (node:internal/modules/esm/resolve:926:18)
    at defaultResolve (node:internal/modules/esm/resolve:1056:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:654:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:603:25)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:586:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:242:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:135:49)
 ELIFECYCLE  Command failed with exit code 2.
```

### PR DESCRIPTION
**Pull request by GrygrFlzr**
Fixes #374
I used version `@eslint/js@^9.17.0` since that's what's installed on our pnpm lockfile for this project, but maybe we can require lower?

### CODE REVIEW

**Comment by manuel3108** [comment]
Mhmm, interesting. I tried reproducing exactly what you did in #374 and for me, it's not failing. This is because `@eslint/js` is a transitive dependency of `eslint`, so I cannot imagine how this case happened
image

**Comment by GrygrFlzr** [comment]
@manuel3108 Thanks for the hint, it looks like it's specifically related to pnpm 10 (which was installed on my system) and I am indeed unable to replicate on pnpm 9. I guess I can file this as a pnpm 10 regression instead? I'll keep the issue open for now in case it's closed as works-as-intended on their end and we've been relying on undefined behavior for pnpm 9 and below.

**Comment by manuel3108** [comment]
@GrygrFlzr Sure, sounds good!

**Comment by GrygrFlzr** [comment]
Per #374 we really *are* using `@eslint/js` as a direct dependency, not a transitive one, and so it should be treated as such. Reopened for review.

Aside, I will also have to investigate if the prettier config generated by the CLI has similar implied dependency issues.

File: .changeset/lemon-cycles-mix.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/addons': patch
+---
+
+Properly add ESLint dependency
```
Comment (benmccann) [inline .changeset/lemon-cycles-mix.md:5]: To match https://github.com/sveltejs/cli/blob/main/packages/cli/CHANGELOG.md
```suggestion
fix: properly add ESLint dependency
```

**Review by benmccann — APPROVED** [review]

File: .changeset/lemon-cycles-mix.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/addons': patch
```
Comment (manuel3108) [inline .changeset/lemon-cycles-mix.md:2]:
```suggestion
'sv': patch
```

We currently only publish `sv` by bundling in the other packages

File: .changeset/lemon-cycles-mix.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'sv': patch
+---
+
+fix: properly add ESLint dependency
```
Comment (manuel3108) [inline .changeset/lemon-cycles-mix.md:5]:
```suggestion
fix: properly add `eslint` dependency
```

**Review by manuel3108 — APPROVED** [review]

**Pull request merged**